modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -55,10 +77,53 @@ function inope_policy_review_tab_title_callback($node) {
  *   policy nodes.
  */
 function inope_policy_review_page($node) {
-  // Create a build array to be displayed on the page.
-  $build = array(
-    'inope_policy_review_form' => drupal_get_form('inope_policy_review_form', $node),
-  );
+  global $user;
+  $account = $user;
+  $build = array();
+  $policy_review_id = NULL;
+
+  // Get the parameters passed in the url and if possible set policy_review_id.
+  $get_params = drupal_get_query_parameters();
+  if (!empty($get_params['id'])) {
+    $policy_review_id = $get_params['id'];
```

Comment:
The general idea with param fetching and arg fetching and anything else is you should check drupal.org to see if there is a drupal function to do what you are doing - because it will likely be doing what you want in a more drupal-y way.  Its the difference between PHP and drupal -- if you get my meaning. PHP is not wrong but see if there is a drupal way because it may have additional considerations or be more future proof for the future of the project.